src/Typeahead.jsx

```diff
@@ -251,7 +270,7 @@ class Typeahead extends React.Component {
    * @param     {Event}    event
    */
   handleKeyDown(event) {
-    if (this.isSpecialKey(event.key)) {
+    if (!this.isSpecialKey(event.key)) {
```

Comment:
there may have been a merge error, when i started these changes my develop branch was not up to date. Im pretty sure we had this taken care of, but it was wrong in my codebase.